src/metabase/driver/generic_sql/query_processor.clj

```diff
@@ -158,17 +176,18 @@
         form
         (recur form more)))))
 
-
 (defn apply-breakout
   "Apply a `breakout` clause to HONEYSQL-FORM. Default implementation of `apply-breakout` for SQL drivers."
-  [_ honeysql-form {breakout-fields :breakout, fields-fields :fields}]
-  (-> honeysql-form
-      ;; Group by all the breakout fields
-      ((partial apply h/group) (map formatted breakout-fields))
-      ;; Add fields form only for fields that weren't specified in :fields clause -- we don't want to include it twice, or HoneySQL will barf
-      ((partial apply h/merge-select) (for [field breakout-fields
-                                            :when (not (contains? (set fields-fields) field))]
-                                        (as (formatted field) field)))))
+  [_ honeysql-form {breakout-fields :breakout, fields-fields :fields :as query}]
+  (as-> honeysql-form new-hsql
+    (apply h/merge-select new-hsql (for [field breakout-fields
+                                         :when (not (contains? (set fields-fields) field))]
+                                     (as (formatted field) field)))
+    (apply h/group new-hsql (map (fn [field]
+                                   (if (instance? BinnedField field)
+                                     (qualified-alias field)
```

Comment:
This is ugly and probably not what we want. I'm not sure if this is done elsewhere, but what I have is a computed projected value that is aliased (via the `as` above) that I want to reference in the group by and the order by. Using formatted in each place (as the code was before) it will repeat that computation in the select, group by and order by. With the above change, the query looks like:

```
SELECT count(*) AS "count",
       (floor(("PUBLIC"."VENUES"."LATITUDE" / 2.0)) * 2.0) AS "LATITUDE",
       (floor(("PUBLIC"."VENUES"."LONGITUDE" / 5.0)) * 5.0) AS "LONGITUDE"
FROM "PUBLIC"."VENUES"
GROUP BY "LATITUDE", "LONGITUDE"
ORDER BY "LATITUDE", "LONGITUDE"
```